test/integration/generated/headerprefix/src/headerPrefixClient.ts

```diff
@@ -39,6 +40,29 @@ export class HeaderPrefixClient extends coreClient.ServiceClient {
       baseUri: options.endpoint ?? options.baseUri ?? "{$host}"
     };
     super(optionsWithDefaults);
+
+    if (options?.pipeline && options.pipeline.getOrderedPolicies().length > 0) {
+      const pipelinePolicies: coreRestPipeline.PipelinePolicy[] = options.pipeline.getOrderedPolicies();
```

Comment:
```suggestion
      const pipelinePolicies= options.pipeline.getOrderedPolicies();
```

Same here 🙂

Reply:
Same as last one since this is generated from the last code